tests/cases/heatmap.js

```diff
@@ -31,6 +31,7 @@ describe('canvas heatmap feature', function () {
   });
 
   it('Setup map', function () {
+    mockAnimationFrame();
```

Comment:
why this change (moving the code to here)? I guess I have an idea but still curious